### PR TITLE
Update to Go 1.17.5 and golang.org/x/net to v0.0.0-20211209124913-491a49abca63

### PR DESCRIPTION
**Pull request by xmudrii**
**What this PR does / why we need it**:

Update to Go 1.17.5 and golang.org/x/net to v0.0.0-20211209124913-491a49abca63.

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

/assign @kron4eg

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -121,6 +121,7 @@ require (
 )
 
 replace (
+	golang.org/x/net => golang.org/x/net v0.0.0-20211209124913-491a49abca63 // CVE-2021-44716
```
Comment (kron4eg) [inline go.mod:124]: it's possible to achieve the same effect without replace directive
```
go get -v -d golang.org/x/net@491a49abca63
```

Comment (xmudrii) [reply go.mod]: Good point, done.

**Review by kron4eg — APPROVED** [review]

**Pull request merged**